spec/spec_helper.rb

```diff
@@ -46,7 +46,7 @@ def new_connection
   when 'mysql'
     password_argument = (PASSWORD.empty?) ? "" : "-p#{PASSWORD}"
     Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "DROP DATABASE IF EXISTS #{DATABASE}" }
-    Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "CREATE DATABASE #{DATABASE} CHARSET utf8" }
+    Kernel.system %{ mysql -h 127.0.0.1 -u #{CURRENT_USER} #{password_argument} -e "CREATE DATABASE #{DATABASE} CHARSET utf8mb4 COLLATE utf8mb4_general_ci" }
```

Comment:
We're replacing `utf8` with `utf8mb4`, which __should__ be the utf8 standard for MySQL moving forward (since sometime in 2010, actually).  The current `utf8` setting for MySQL actually only supports a very small subset of the real UTF-8 bit set of characters (something like 2%, from one of the blog posts I read while banging my head against the wall trying to get this sorted).  I think a `README` note about using `utf8mb4` might be more appropriate, rather than attempting to support something that's essentially broken.

Your call, though.  I can add a `MYSQL_OLD` setting to the build matrix if you'd like :)